students/ZidanLuo/FizzBuzzTest.py

```diff
@@ -0,0 +1,13 @@
+def FizzBuzz():
+	for i in range(1, 101):
+		if i % 3 == 0:
```

Comment:
can you clean this up to be a little less redundant? 

<hint>

if you do:

`print("something",  end='')`

you won't get a newline after the "something".